README.md

```diff
@@ -88,7 +101,405 @@ workflow wf {
 }
 ```
 
-Architecture
-------------
+# Getting Started with WDL
+
+We'l use Cromwell to run these examples but you can use any WDL engine of your choice. If you don't already have a reference to the Cromwell JAR file, one can be [downloaded](https://github.com/broadinstitute/cromwell/releases) or [built manually](#building)
+
+## Hello World WDL
+
+Create a WDL simple file and save it as `hello.wdl`, for example:
+
+```
+task hello {
+  String name
+
+  command {
+    echo 'hello ${name}!'
+  }
+  output {
+    File response = stdout()
+  }
+}
+
+workflow test {
+  call hello
+}
+```
+
+Generate a template `hello.json` file with the `inputs` subcommand:
+
+```
+$ java -jar cromwell.jar inputs hello.wdl
```

Comment:
For now, yes. Note his is only the library portion and not he clp. That'll be later